Added required functionality for Sprint_4 tests

Sergey8193 opened the pull request:
Added
1. New order creation test
2. FAQ answers relevance test
3. Click logo scooter test
4. Click logo Yandex test
5. Track non existent order test
6. New order input errors test

egorvas commented on src/main/java/ru/yandex/praktikum/qa_scooter/page_objects/PageHeader.java:
```diff
@@ -0,0 +1,84 @@
+package ru.yandex.praktikum.qa_scooter.page_objects;
```
**Можно лучше:** в названиях пекаджей не принято испрользовать _

Sergey8193 replied:
qa_scooter -> qascooter
page_objects -> pageobjects

egorvas commented on src/test/java/ru/yandex/praktikum/qa_scooter/NewOrderTest.java:
```diff
@@ -0,0 +1,163 @@
+    }
+
+    @Test
+    public void startTests() {
```
**Нужно исправить:** совершенно не понимаю в чем смысл этого. Ты же можешь просто своим функциям newOrderCreationTest и newOrderInputFieldErrorMessagesTest сделать аннотацию @Test и они будут отдельными тестами.
Такая реализация как у тебя явно избыточна

Sergey8193 replied:
@Test                                    @Test                               @Test
public void startTests()   ->   newOrderCreationTest()   newOrderInputFieldErrorMessagesTest()

egorvas commented on src/main/java/ru/yandex/praktikum/qa_scooter/constants/Durations.java:
```diff
@@ -0,0 +1,14 @@
+    public static final Duration ONE_SECOND = Duration.ofSeconds(1);
+    public static final Duration THREE_SECONDS = Duration.ofSeconds(3);
+    public static final Duration FIVE_SECONDS = Duration.ofSeconds(5);
+    public static final Duration TEN_SECONDS = Duration.ofSeconds(10);
```
**Можно лучше**: выносить цифры в константы это что за гранью, явно избыточно.

Sergey8193 replied:
Перешел к следующей классификации задержек

    public static final Duration SCOOTER_PAGE_TIMEOUT = Duration.ofSeconds(5);
    public static final Duration SCOOTER_PAGE_SECTION_TIMEOUT = Duration.ofSeconds(3);
    public static final Duration SCOOTER_PAGE_ELEMENT_TIMEOUT = Duration.ofSeconds(1);
    public static final Duration YANDEX_PAGE_TIMEOUT = Duration.ofSeconds(5);

egorvas commented on src/test/java/ru/yandex/praktikum/qa_scooter/TrackPageTest.java:
```diff
@@ -0,0 +1,71 @@
+    }
+
+    @Test
+    public void OrderStatusTest() {
```
**Нужно исправить:** функции java всегда с маленбкой буквы

Sergey8193 replied:
Спасибо, мой недосмотр: OrderStatusTest -> оrderStatusTest

Sergey8193 commented:
**Fixed bugs specified during sprint-4 pull request processing:**

**1. Changed Durations classification:**

public static final Duration SCOOTER_PAGE_TIMEOUT = Duration.ofSeconds(6);
public static final Duration SCOOTER_PAGE_SECTION_TIMEOUT = Duration.ofSeconds(4);
public static final Duration SCOOTER_PAGE_ELEMENT_TIMEOUT = Duration.ofSeconds(2);
public static final Duration YANDEX_PAGE_TIMEOUT = Duration.ofSeconds(6);

**2. Changed packages names:**

qa_scooter -> qascooter
page_objects -> pageobjects

**3. Removed excess methods**

@test
public void startTests(); ->

@test
public void newOrderCreationTest();

@test
newOrderInputFieldErrorMessagesTest();

**4. Fixed typos:**

OrderStatusTest() -> orderStatusTest()
